integrations/docker/images/chip-build-nrf-platform/Dockerfile

```diff
@@ -35,7 +35,7 @@ ENV ARM_GCC_INSTALL_ROOT=/opt/ARM-software/gcc-arm-none-eabi-9-2019-q4-major/bin
 # ==================================================
 RUN set -x \
     # Device Tree Compiler 1.4.7
-    && curl -o /tmp/dtc.deb http://mirrors.edge.kernel.org/ubuntu/pool/main/d/device-tree-compiler/device-tree-compiler_1.4.7-3ubuntu2_amd64.deb \
+    && curl -o /tmp/dtc.deb http://ftp.br.debian.org/debian/pool/main/d/device-tree-compiler/device-tree-compiler_1.4.7-3_amd64.deb \
```

Comment:
Why the .br. site? Is there something that's more edge cached?

Reply:
Oh, you're right. I took download link from [debian.pkgs.org](https://debian.pkgs.org/10/debian-main-amd64/device-tree-compiler_1.4.7-3_amd64.deb.html) but I didn't even notice that .br. part of site name.
Ok, I see that we can use [mirrors.edge.kernel.org/debian/](https://mirrors.edge.kernel.org/debian/pool/main/d/device-tree-compiler/device-tree-compiler_1.4.7-3_amd64.deb) for download.

Reply:
Done.

Reply:
I removed this change due to #4657 .